packages/devtools_app/pubspec.yaml

```diff
@@ -56,6 +56,8 @@ dependencies:
   vm_service: ^7.1.1
   vm_snapshot_analysis: ^0.6.0
   web_socket_channel: ^2.1.0
+  widget_icons:
+    path: ../../third_party/packages/widget_icons
```

Comment:
Actually, just write `widget_icons: ^0.0.1` and comment it out, since we aren't actually depending on the widget_icons package at all in this PR. Then we can publish and uncomment once this PR lands.

Reply:
Makes sense, I commented it out.